frontend/src/components/Workspace/FlowChart/Dialog/FileSelectDialog.tsx

```diff
@@ -300,7 +300,7 @@ const TreeItemLabel = memo(function TreeItemLabel({
       <Box>
         <Checkbox
           {...checkboxProps}
-          disableRipple
+          // disableRipple
```

Comment:
Could you tell me the reason for this change?

Reply:
@ReiHashimoto Sorry about this problem, I commented again to test but forgot to rollback